recipes/pip/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Yeah, it uses `setuptools`.

So let's add the arguments `--single-version-externally-managed --record record.txt`.